pkg/controller/operators/catalog/operator.go

```diff
@@ -77,6 +77,8 @@ const (
 	maxInstallPlanCount    = 5
 	maxDeletesPerSweep     = 5
 	RegistryFieldManager   = "olm.registry"
+	// TODO(haseeb): Change to a more realistic timeout for bundle unpack jobs
+	DefaultBundleUnpackTimeout = 2 * time.Minute
```

Comment:
My impression is that 5 minutes might be a more reasonable deadline, if we consider how long it takes to pull a UBI based image, extract all its contents, and put the manifests we care about into a configmap. 

Might just be worth seeing how long it takes for the official red hat operators catalog (without that image already present on the node) and setting it around that

Reply:
I've definitely spoken with users who have had unpacking take over 10 minutes. I think ultimately we want to give up fast when there are immediate errors (like `opm` crashing repeatedly), but allow more time when there may be nothing wrong other than slow I/O.

Reply:
To aid us in failing fast for immediate crashing errors we already have the `spec.backoffLimit` field right now which we don't set but by default is set to 6.
https://kubernetes.io/docs/concepts/workloads/controllers/job/#pod-backoff-failure-policy
```
Failed Pods associated with the Job are recreated by the Job controller with an exponential back-off delay (10s, 20s, 40s ...) capped at six minutes.
```

Although given that it will take `10s + 20s + 40s + 80s + 160s +320s = 10.5 minutes` that's probably not fast enough. So maybe `spec.backoffLimit=4` or `2.5m` gets us closer to failing fast for repeated pod failures.

Reply:
I think I also need to set the pod failure policy `restartPolicy = "Never"` so the job can spawn a new pod per failure. Otherwise the failed containers go away on restart and it'll be hard to inspect failure logs.